src/components/NavBar/NavbarLoggedOut.jsx

```diff
@@ -40,9 +40,10 @@ const NavbarLoggedOut = () => {
             <>
               <Button
                 variant="contained"
-                color="tertiary"
+                color="secondary"
                 size="large"
-                sx={{ opacity: '0.8', color: 'black' }}
+                // sx={{ opacity: '0.8', color: 'black' }}
```

Comment:
Think we can remove this `sx` prop if we're going to use the styles below with the border radius.

Reply:
ahhh, missed it. Thanks!